typescript/sdk/src/hook/types.ts

```diff
@@ -27,6 +27,7 @@ export enum OnchainHookType {
 }
 
 export enum HookType {
+  ADDRESS = 'address',
```

Comment:
can we be consistent with IsmType? either ADDRESS or CUSTOM in both?

Reply:
fixed